src/groups/mwc/mwcc/mwcc_orderedhashmap.t.cpp

```diff
@@ -276,13 +276,13 @@ static void test4_rinsert()
 
 #ifdef BSLS_PLATFORM_OS_AIX
     // Avoid timeout on AIX
-    const size_t k_NUM_ELEMENTS = 100 * 1000;  // 100K
+    const int k_NUM_ELEMENTS = 100 * 1000;  // 100K
 #else
-    const size_t k_NUM_ELEMENTS = 1000 * 1000;  // 1M
+    const int    k_NUM_ELEMENTS = 1000 * 1000;  // 1M
```

Comment:
```suggestion
    const int k_NUM_ELEMENTS = 1000 * 1000;  // 1M
```

Reply:
Yeah, this formatting looks strange, but that's exactly what clang-format did. CI fails otherwise, check the last issue in this [log](https://github.com/bloomberg/blazingmq/actions/runs/9093692307/job/24993176983?pr=281)